packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/camera.dart

```diff
@@ -110,42 +110,59 @@ class CameraPosition {
       'CameraPosition(bearing: $bearing, target: $target, tilt: $tilt, zoom: $zoom)';
 }
 
+/// Indicates which type of camera update this instance represents.
+enum CameraUpdateType {
```

Comment:
Ideally we would make CameraUpdate a sealed class so that we could just do exhaustive checked switches over the classes themselves, but since that would technically be a breaking change this works.